src/status_im2/contexts/profile/login/events.cljs

```diff
@@ -0,0 +1,3 @@
+(ns status-im2.contexts.profile.login.events)
+
```

Comment:
is this WIP?

Reply:
yes, there will be next parts, this PR is part1